pkg/util/test/tls/kms_tls_test.go

```diff
@@ -0,0 +1,72 @@
+package kms_tls_test
+
+import (
+	"os"
+
+	nbv1 "github.com/noobaa/noobaa-operator/v5/pkg/apis/noobaa/v1alpha1"
+	"github.com/noobaa/noobaa-operator/v5/pkg/options"
+	"github.com/noobaa/noobaa-operator/v5/pkg/system"
+	"github.com/noobaa/noobaa-operator/v5/pkg/util"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func getMiniNooBaa() *nbv1.NooBaa {
+	options.MiniEnv = true
+	options.Namespace = corev1.NamespaceDefault
+	nb := system.LoadSystemDefaults()
+	return nb
+}
+
+func tlsSAKMSSpec(api_address string) nbv1.KeyManagementServiceSpec {
```

Comment:
missing here test for tls when using token auth, and also putting here vaultSkipVerify true is actually not testing the tls part

Reply:
✅ See [commit](https://github.com/noobaa/noobaa-operator/pull/797/commits/fccc87b78ea5f7d3b41bc55c73621dd5b2c307e3)